hedera-node/data/config/application.properties

```diff
@@ -0,0 +1,16 @@
+bootstrap.throttleDefsJson.resource=throttles-dev.json
+ledger.id=0x03
+staking.periodMins=1
+scheduling.whitelist=ConsensusSubmitMessage,CryptoTransfer,TokenMint,TokenBurn,CryptoApproveAllowance,CryptoUpdate
+# Uncomment a line below to test HAPI operations via workflows
+hedera.workflows.enabled=true
+#hedera.workflows.enabled=ConsensusCreateTopic,ConsensusUpdateTopic,ConsensusDeleteTopic,ConsensusSubmitMessage,ConsensusGetTopicInfo
```

Comment:
Nit: do we need this comment in here any more? Maybe it Is left here just to show examples of how to enable workflows on an individual class basis ?

Reply:
Good catch. I am not even sure if this still works. This file needs some cleanup. We should do this in a follow-up PR.

Reply:
Agreed! Will change in next PR. Thanks @thomas-swirlds-labs @netopyr